projects/igniteui-angular/src/lib/icon/types.ts

```diff
@@ -23,6 +23,7 @@ export interface IconMeta {
     name: string;
     family: string;
     type?: IconType;
+    external?: boolean;
```

Comment:
Any chance we can keep this on the internal types only?

Reply:
This has also been exposed like that in the Web Components registry.